docs/topics/db/optimization.txt

```diff
@@ -261,46 +261,40 @@ Don't overuse ``count()`` and ``exists()``
 If you are going to need other data from the QuerySet, evaluate it immediately.
 
 For example, assuming an Email model that has a ``body`` attribute and a
-many-to-many relation to User, the following template code is optimal:
+many-to-many relation to User, the following code is optimal:
 
-.. code-block:: html+django
-
-   {% if display_inbox %}
-     {% with emails=user.emails.all %}
-       {% if emails %}
-         <p>You have {{ emails|length }} email(s)</p>
-         {% for email in emails %}
-           <p>{{ email.body }}</p>
-         {% endfor %}
-       {% else %}
-         <p>No messages today.</p>
-       {% endif %}
-     {% endwith %}
-   {% endif %}
+.. code-block:: python
```

Comment:
This is unnecessary, use `... is optimal::` instead.

Reply:
I prefer the explicit style, it's TOOWTDI and easier for new users of rST to grok. But will change to match the rest of the docs